src/features/surveys/components/surveyForm/SurveySignature.tsx

```diff
@@ -0,0 +1,139 @@
+import messageIds from 'features/surveys/l10n/messageIds';
+import { Msg } from 'core/i18n';
+import SurveyContainer from './SurveyContainer';
+import SurveyOption from './SurveyOption';
+import SurveySubheading from './SurveySubheading';
+// import useCurrentUser from 'features/user/hooks/useCurrentUser';
+
+import {
+  Box,
+  FormControl,
+  FormLabel,
+  Radio,
+  RadioGroup,
+  TextField,
+  Typography,
+  useTheme,
+} from '@mui/material';
+import { FC, useCallback, useState } from 'react';
+import {
+  ZetkinSurveyExtended,
+  ZetkinSurveySignatureType,
+  ZetkinUser,
+} from 'utils/types/zetkin';
+
+export type SurveySignatureProps = {
+  survey: ZetkinSurveyExtended;
+  user: ZetkinUser | null;
+};
+
+const SurveySignature: FC<SurveySignatureProps> = ({ survey, user }) => {
+  // const currentUser = useCurrentUser();
```

Comment:
Why is this currently commented out? It looks to me like the user is coming in through props, so I'm assuming this line could be deleted?